main.go

```diff
@@ -50,24 +50,29 @@ func main() {
 	var owner = flag.String(
 		"owner",
 		"",
-		"The owner, e.g. your-name")
+		"The owner of the repository, e.g. it is 'atlassian' in 'https://bitbucket.org/atlassian/stash-example-plugin/src/master/'")
 
 	var repositoryName = flag.String(
 		"repositoryName",
 		"",
-		"The repositoryName, e.g. some-repository")
+		"The name of the repository, e.g. it is 'stash-example-plugin' in 'https://bitbucket.org/atlassian/stash-example-plugin/src/master/'")
 
 	var buildNumber = flag.String(
 		"buildNumber",
 		"",
-		"The buildNumber, e.g. buildNumber")
+		"The build number, e.g. the unique identifier of a build e.g. 333")
 
 	var buildURL = flag.String(
 		"buildURL",
 		"",
-		"The buildURL, e.g. buildURL")
+		"The url of the specific build (so bitbucket can direct you back), e.g. https://travis-ci.org/030/golang-bitbucket-cloud-build-status-notifier/jobs/523263435")
 
-	utils.Debug()
+	flag.Parse()
+
+	if *keyString == "" || *buildState == "" || *gitCommit == "" || *owner == "" || *repositoryName == "" || *buildNumber == "" || *buildURL == "" {
```

Comment:
Perhaps use required Flags, see:

 	// Example of a required flag
	Name string `short:"n" long:"name" description:"A name" required:"true"`

https://stackoverflow.com/a/40829034/2777965

Reply:
This uses another package i.e. not the built-in flags package I'm using here, and if we are using a different library for the flags I'd rather use [Cobra](https://github.com/spf13/cobra)

Reply:
that is fine, see GH-25